src/client/auth.ts

```diff
@@ -146,7 +153,7 @@ export async function auth(
     metadata,
     clientInformation,
     redirectUrl: provider.redirectUrl,
-    scope: provider.clientMetadata.scope
+    scope,
```

Comment:
```suggestion
    scope: scope || provider.clientMetadata.scope,
```
My thinking is that if someone has put scope in their clientMetadata, but not provided it in auth, they likely want to use those scopes rather than leave it blank.

also, this will avoid making this a breaking change.  (That's relatively low stakes since the scope passing was just added last week, but still nice to not be breaking).

Reply:
Thanks for the suggestion! That makes a lot of sense. I just updated the code, PTAL.